example.js

```diff
@@ -15,9 +18,9 @@ app.router([
     done(error(500, 'server error!'))
   }],
   ['/api', {
-    get: function (req, res, ctx, done) {
+    'get': cors(function (req, res, ctx, done) {
```

Comment:
think we should just make this a `/cors` route on the example; also no need to quote